google-cloud-clients/google-cloud-bigtable-admin/README.md

```diff
@@ -0,0 +1,134 @@
+# Google Cloud Java Client for Bigtable Admin
+
+Java idiomatic client for [Cloud Bigtable Admin][cloud-bigtable]. Please note that this client is under
+heavy development and is not ready for production use. Please continue to use the 
+[HBase API client](https://github.com/GoogleCloudPlatform/cloud-bigtable-client) for production.
+
+[[![CircleCI](https://circleci.com/gh/GoogleCloudPlatform/google-cloud-java/tree/master.svg?style=shield)](https://circleci.com/gh/GoogleCloudPlatform/google-cloud-java/tree/master)
+[![Coverage Status](https://coveralls.io/repos/GoogleCloudPlatform/google-cloud-java/badge.svg?branch=master)](https://coveralls.io/r/GoogleCloudPlatform/google-cloud-java?branch=master)
+[![Maven](https://img.shields.io/maven-central/v/com.google.cloud/google-cloud-bigtable.svg)](https://img.shields.io/maven-central/v/com.google.cloud/google-cloud-bigtable-admin.svg)
+[![Codacy Badge](https://api.codacy.com/project/badge/grade/9da006ad7c3a4fe1abd142e77c003917)](https://www.codacy.com/app/mziccard/google-cloud-java)
+[![Dependency Status](https://www.versioneye.com/user/projects/58fe4c8d6ac171426c414772/badge.svg?style=flat)](https://www.versioneye.com/user/projects/58fe4c8d6ac171426c414772)
+
+- [Product Documentation][bigtable-product-docs]
+- [Client Library Documentation][bigtable-client-lib-docs]
+
+> Note: This client is under heavy development and should not be used in production.
+
+## Quickstart
+
+[//]: # ({x-version-update-start:google-cloud-bigtable:released})
```

Comment:
this needs to be `google-cloud-bigtable-admin` so it gets the right version

Reply:
done